cli/cmd/vuln_container.go

```diff
@@ -65,6 +70,50 @@ Arguments:
 		},
 	}
 
+	// vulContainerListRegistriesCmd represents the list-registries sub-command inside the container
+	// vulnerability command
+	vulContainerListRegistriesCmd = &cobra.Command{
+		Use:     "list-registries",
+		Aliases: []string{"list-reg", "registries"},
+		Short:   "list all Container Registries configured",
+		Long:    `List all Container Registries configured in your account.`,
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, args []string) error {
+			registries, err := getContainerRegistries()
+			if err != nil {
+				return err
+			}
+			if len(registries) == 0 {
+				msg := `There are no Container Registries configured in your account.
+
+Get started by integrating your Container Registry using the command:
+
+    $ lacework integration create
+
+Or, if you prefer to do it via the WebUI, log in to your account at:
```

Comment:
```suggestion
If you prefer to configure the integration via the WebUI. Log in to your account at:
```

Reply:
@afiune I see we use that language in other areas in the CLI. I would like @dmurray-lacework to weigh in here too, but starting a sentence with "Or, ..." is not proper. A comma would typically precede "Do it this way, OR that way."

If we are all in agreement, then we should fix that wording everywhere.

Reply:
Thank you! Thank you so much, my friend!! I will update all the locations where we have this sentence:
```
cli/cmd/compliance.go:Or, if you prefer to do it via the WebUI, log in to your account at:
cli/cmd/compliance_aws.go:Or, if you prefer to do it via the WebUI, log in to your account at:
cli/cmd/compliance_azure.go:Or, if you prefer to do it via the WebUI, log in to your account at:
cli/cmd/vuln_container.go:Or, if you prefer to do it via the WebUI, log in to your account at:
cli/cmd/vuln_container.go:Or, if you prefer to do it via the WebUI, log in to your account at:
cli/cmd/vulnerability.go:Or, if you prefer to do it via the WebUI, log in to your account at:
cli/docs/lacework_compliance.md:Or, if you prefer to do it via the WebUI, log in to your account at:
cli/docs/lacework_vulnerability_container.md:Or, if you prefer to do it via the WebUI, log in to your account at:
integration/compliance_test.go:Or, if you prefer to do it via the WebUI, log in to your account at:
```

Reply:
![tenor-70325890](https://user-images.githubusercontent.com/5712253/112046207-58799500-8b11-11eb-8ce1-9052391760aa.gif)